<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bookDialog</class>
 <widget class="QDialog" name="bookDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>0</y>
     <width>291</width>
     <height>531</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="horizontalSpacing">
     <number>3</number>
    </property>
    <property name="verticalSpacing">
     <number>3</number>
    </property>
    <item row="3" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>书籍编号：</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="bidLine">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>书籍名称：</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="nameLine"/>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>书籍作者：</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="authoLine"/>
    </item>
    <item row="18" column="0" colspan="2">
     <widget class="QPushButton" name="pushButton">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>修改信息</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLabel" name="label">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>书籍信息修改</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>书籍出版社：</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QLineEdit" name="pressLine"/>
    </item>
    <item row="14" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>书籍库存：</string>
      </property>
     </widget>
    </item>
    <item row="14" column="1">
     <widget class="QLineEdit" name="stockLine"/>
    </item>
    <item row="10" column="0" colspan="2">
     <spacer name="verticalSpacer_5">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="8" column="0" colspan="2">
     <spacer name="verticalSpacer_4">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="6" column="0" colspan="2">
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="0" colspan="2">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0" colspan="2">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="16" column="0" colspan="2">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>修改图片</string>
      </property>
     </widget>
    </item>
    <item row="17" column="0" colspan="2">
     <spacer name="verticalSpacer_6">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="15" column="0" colspan="2">
     <spacer name="verticalSpacer_7">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="12" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>书籍类别：</string>
      </property>
     </widget>
    </item>
    <item row="12" column="1">
     <widget class="QLineEdit" name="typeLine"/>
    </item>
    <item row="13" column="0" colspan="2">
     <spacer name="verticalSpacer_8">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
